src/DIRAC/Core/scripts/dirac_apptainer_exec.py

```diff
@@ -0,0 +1,84 @@
+""" Starts a DIRAC command inside an apptainer container.
```

Comment:
This is a great idea!

Reply:
I pushed this the other day but have yet to do a single test with it.

Anyway, a use case for this is clearly running, from the pilot, the `dirac-platfom` (`dirac-architecture`) script.